CI: disable SYCL warp tests

psychocoderHPC opened the pull request:
The warp tests in alpaka assume that warp functions are not collective.
In #2470 we missed to deactivate the tests disabled in this PR.

This PR is required for the fix of the SYCL index order #2488

fwyzard reviewed (dismissed):
The PR includes whitespace-only changes to unrelated files.

I'm find to include them here, I'm pointing it out just to make sure it was intended.

SimeonEhrig commented on script/readme_generator/generate_supported_compilers.py:
```diff
@@ -209,18 +204,10 @@ def render_table(conf) -> str:
-                "Generate a new table with "
-                "'./script/readme_generator/generate_supported_compilers.py'"
-            )
+            print("Please the check the configuration file " "'./script/readme_generator/supported_compilers.json'")
```
```suggestion
            print("Please the check the configuration file './script/readme_generator/supported_compilers.json'")
```

SimeonEhrig commented on script/readme_generator/generate_supported_compilers.py:
```diff
@@ -209,18 +204,10 @@ def render_table(conf) -> str:
-                "'./script/readme_generator/generate_supported_compilers.py'"
-            )
+            print("Please the check the configuration file " "'./script/readme_generator/supported_compilers.json'")
+            print("Generate a new table with " "'./script/readme_generator/generate_supported_compilers.py'")
```
```suggestion
            print("Generate a new table with './script/readme_generator/generate_supported_compilers.py'")
```

SimeonEhrig commented on script/readme_generator/generate_supported_compilers.py:
```diff
@@ -209,18 +204,10 @@ def render_table(conf) -> str:
-                "Verify the README.md with "
-                "'./script/readme_generator/generate_supported_compilers.py "
-                "--verify'"
+                "Verify the README.md with " "'./script/readme_generator/generate_supported_compilers.py " "--verify'"
```
```suggestion
                "Verify the README.md with './script/readme_generator/generate_supported_compilers.py --verify'"
```

SimeonEhrig commented on script/readme_generator/generate_supported_compilers.py:
```diff
@@ -135,13 +133,10 @@ def render_table(conf) -> str:
     parser = argparse.ArgumentParser(
-        description="Creates a Markdown table for the supported compilers from "
-        "the configuration file."
+        description="Creates a Markdown table for the supported compilers from " "the configuration file."
```
```suggestion
        description="Creates a Markdown table for the supported compilers from the configuration file."
```

SimeonEhrig commented:
@psychocoderHPC Did you changed something on the python code except formatting?

psychocoderHPC commented:
sry was running pre commits with EOF white space remove. Will fix it.

psychocoderHPC committed:
CI: disable SYCL warp tests

The warp tests in alpaka assume that warp functions are not collective.
In #2470 we missed to deactivate the tests disabled in this PR.

psychocoderHPC commented:
@SimeonEhrig @fwyzard I fixed the PR, it includes now only the disabled tests, sry for the wrong changes in the first version.
